postgres/pyproject.toml

```diff
@@ -37,10 +37,12 @@ license = "BSD-3-Clause"
 
 [project.optional-dependencies]
 deps = [
-    "boto3==1.27.0; python_version > '3.0'",
+    "boto3==1.17.112; python_version < '3.0'",
```

Comment:
You could simply drop this line as this check no longer support py2

We could create a card to add that to the ddev command

Reply:
Created [AITOOLS-237](https://datadoghq.atlassian.net/browse/AITOOLS-237)